functions/daemon.go

```diff
@@ -173,8 +173,13 @@ func startGoRoutines(wg *sync.WaitGroup) context.CancelFunc {
 	}
 	if err := nc.Configure(); err != nil {
 		slog.Error("error configuring netclient interface", "error", err)
-		nc.Close()
-		os.Exit(1)
+		// needed to prevent exit when there are no peers
+		if err.Error() == "ioctl: bad address" {
```

Comment:
looking at this, I am worried that maybe there are other errors that we don't know about that shouldn't be a fatal here

Reply:
we were always exiting with fatal error previously